scene/gui/tree.cpp

```diff
@@ -1258,6 +1258,16 @@ void TreeItem::deselect(int p_column) {
 	_cell_deselected(p_column);
 }
 
+void TreeItem::clear_buttons() {
+	for (int i = 0; i < cells.size(); ++i) {
```

Comment:
```suggestion
	for (Cell &cell : cells) {
```

Reply:
Sure, I just did what surrounding code did, I'll change this.

Reply:
You can track index separately.
```C++
int i = 0;
for (Cell &cell : cells) {
	...stuff
	i++;
}

Reply:
We could, but is that more readable than a for loop?

Reply:
It's probably less readable, but should be a bit faster.

Reply:
alright, done!